sphinx_gallery/gen_gallery.py

```diff
@@ -1138,7 +1138,7 @@ def _make_graph(fname, entries, gallery_conf):
                 dg.node(node_to, **node_kwargs)
                 dg.edge(node_from, node_to, color=API_COLORS["edge"])
         # add modules with all API entries
-        for module in gallery_conf["_sg_api_entries"]["module"]:
+        for module in gallery_conf["_sg_api_entries"].get("module", []):
```

Comment:
Every once in a while when rebuilding MNE-Python doc I'd get an error on this line. No idea why and couldn't figure out how to reproduce it, but it should be safe enough to use `.get` instead of direct dict access here.